spec-tests/src/test/java/org/candlepin/spec/owners/OwnerResourceSpecTest.java

```diff
@@ -792,4 +795,41 @@ public void shouldAllowUserWithOwnerPoolsPermissionCanSeeSystemPurposeOfTheOwner
         assertEquals(owner.getKey(), attributes.getOwner().getKey());
     }
 
+    @Test
+    @OnlyInHosted
+    public void shouldMigrateAllConsumersOfAnonymousOwnerToTargetOwner() {
+        String accountId = StringUtil.random("cloud-account-id-");
+        String instanceId = StringUtil.random("cloud-instance-id-");
+        String offerId = StringUtil.random("cloud-offer-");
+        OwnerDTO anonOwner = admin.owners().createOwner(Owners.random().anonymous(true));
```

Comment:
instead of `admin.owners()` you can use the existing `owners` field here, like the line right below this one

Reply:
Done.